src/Home.js

```diff
@@ -159,23 +176,7 @@ const Home = props => {
 
 		const total = '$' + Object.values(filtered).reduce((t, {estimate}) => t + parseFloat(estimate), 0).toFixed(2);
 	
-		const SortIcon = (props) => {
-			return (
-				<span aria-hidden="true">
-				{ props.props === sort
-					? <img src="images/sort.svg" className={`${dir === 'asc' ? styles.asc : styles.desc}`} alt="" />
-					: ''
-				}
-				</span>
-
-			);
-		};
 
-		const SortHeading = (props) => {
-			return (
-				<th aria-sort={sort === props.id ? dir : 'none'} className={props.showmodbile === 'false'? styles.showDesktop : ''} onClick={() => sortItem(props.id)}><button aria-label={props.arialabel} className={styles.clickDisabled} onClick={() => sortItem(props.id)}>{props.name}<SortIcon props={props.id} /></button></th>
-			);
-		}
```

Comment:
having these inside of the function `Home` component, meant for every re-render, `SortIcon` and `SortHeading` was getting re-defined as a brand new component, and react was mounting the brand new component thus losing focus